core/pom.xml

```diff
@@ -194,6 +194,11 @@
             <artifactId>jna</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
I thought this was specifically for Guava dependencies. Are we going to replace every Guava functionality by our own implementation?